docker/psi4/Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM conda/miniconda3:latest
 
 RUN apt-get update && \
-  apt-get install -y gcc && \
+  apt-get install -y gcc git && \
   rm -rf /var/lib/apt/lists/*
 
 RUN conda update --all && \
   conda install psi4=1.2 psi4-rt=1.2 python=3.6 -c psi4
 
-ENTRYPOINT ["/usr/local/bin/psi4"]
+RUN pip install avogadro
```

Comment:
Why not just install all of these in one pip install invocation?

Reply:
Or better yet, use a `requirements.txt` file

Reply:
Or `setup.py`

Reply:
As it is just a list of requirements I think requirements.txt sounds better, it would be more approachable to others to then adapt to their own workflows too.